source/includes/mongodump-restore-version-link.rst

```diff
@@ -1,6 +1,6 @@
 You can restore the BSON files generated from ``mongodump`` into MongoDB
-deployments running the same or newer version as the source deployment.
-However, restoring files into a newer version deployment is not the
-recommended way to upgrade your deployment. To learn how to upgrade your
-deployment, see the :manual:`upgrade documentation
+deployments running the same major version or feature compatibility version as 
+the source deployment. However, restoring files into a newer version 
+deployment is not the recommended way to upgrade your deployment. To learn 
```

Comment:
I think we should probably cut the last two sentences of this paragraph (starting with "however"). Since the support is now restricted to same major version or same FCV, I don't think we need to go into upgrading.